src/commands/config.ts

```diff
@@ -0,0 +1,85 @@
+import { container } from 'tsyringe'
+import {
+  GetPreferenceUseCase,
+  SetPreferenceUseCase,
+  possiblePreferences,
+  Preference,
+  InvalidValueForPreferenceError,
+} from '../modules/preferences'
```

Comment:
Oclif doesn't want to work along with TypeScript path mappings. Which is very weird, since I didn't find any mention to this in the docs. In the meanwhile, `tsc` isn't complaining and the path mappings work fine in test environment. This is something I might look into later.